ui/src/components/ConfigurationView.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { Grid, Row, Col } from 'patternfly-react';
+import Header from './Header';
+import ServiceSDKInfo from './ServiceSDKInfo';
+import InlineEdit from './InlineEdit';
+import CopyToClipboardMultiline from './CopyToClipboardMultiline';
+
+const config = `{
+  "version": 1,
+  "clusterName": "https://192.168.0.106:8443",
+  "namespace": "myproject",
+  "clientId": "myapp-android",
+  "services": []
+}`;
+
+class ConfigurationView extends Component {
+  render() {
+    return (
+      <Grid fluid>
+        <Row>
+          <Col md={6}>
+            <Header>Mobile Client Details</Header>
+            <dl className="dl-horizontal left">
+              <dt>Client Type:</dt>
+              <dd>android</dd>
+              <dt>Client ID:</dt>
+              <dd>com.aerogear.androidshowcase</dd>
+              <dt>Client API Key:</dt>
+              <dd>b331aa8c-6c39-5f07-81f8-9eccf7fd6702</dd>
+              <dt>DMZ Url:</dt>
+              <dd><InlineEdit placeholder="Set DMZ Url" noHorizontalPadding /></dd>
+            </dl>
+          </Col>
+          <Col md={6}>
+            <Header>Mobile Client Config</Header>
+            <CopyToClipboardMultiline style={{ marginBottom: 25 }}>
+              {config}
+            </CopyToClipboardMultiline>
+          </Col>
+        </Row>
+        <Row>
+          <Col md={12}>
+            <Header>SDK Configuration</Header>
+            <h4><a>Android SDK Setup</a></h4>
+          </Col>
+          <Col md={12} style={{ padding: 0 }}>
+            <Header style={{ margin: 20 }}>Service Configuration</Header>
+            <ServiceSDKInfo
+              serviceLogoUrl="https://pbs.twimg.com/profile_images/702119821979344897/oAC05cEB_400x400.png"
```

Comment:
I am not sure of the requirements of this ticker, but should we be passing an array of service info and maybe mapping over it to build out the service sdk info.